dev-tools/formatters/intellij-formatter.xml

```diff
@@ -0,0 +1,77 @@
+<code_scheme name="OpenSearch" version="173">
+  <option name="FORMATTER_ON_TAG" value="end::" />
+  <option name="FORMATTER_OFF_TAG" value="tag::" />
+  <JavaCodeStyleSettings>
+    <option name="CLASS_NAMES_IN_JAVADOC" value="3" />
+    <option name="USE_FQ_CLASS_NAMES" value="true" />
+    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
+    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
+    <option name="IMPORT_LAYOUT_TABLE">
+      <value>
+        <package name="java" withSubpackages="true" static="false" />
+        <package name="javax" withSubpackages="true" static="false" />
+        <emptyLine />
+        <package name="io.opentelemetry" withSubpackages="true" static="false" />
+        <package name="com.google" withSubpackages="true" static="false" />
+        <package name="com.fasterxml" withSubpackages="true" static="false" />
+        <package name="org.apache" withSubpackages="true" static="false" />
+        <package name="org.hamcrest" withSubpackages="true" static="false" />
+        <package name="org.junit" withSubpackages="true" static="false" />
```

Comment:
Why are these called out separately?  Why are there two places that I have to check for 3rd party imports?  I can understand keeping migrations/opensearch things in one block and java stuff in another.  Beyond that, it seems like everything is on equal standing.

Reply:
This list is derived from the existing spotless rules, lets handle that out of band of this change. For `intellij-formatter.xml` was generated by exporting the settings from IntelliJ rather than edited by hand.